src/server/seeds/dev/006_00_vessels.js

```diff
@@ -0,0 +1,28 @@
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex("vessels")
+    .del()
+    .then(function() {
+      // Inserts seed entries
+      return knex("vessels").insert([
+        {
+          id: 1,
+          organization_id: 1,
+          mmsi: 23.33,
+          name: "hyf-vessel1"
+        },
+        {
+          id: 2,
+          organization_id: 1,
+          mmsi: 45.33,
+          name: "hyf-vessel2"
+        },
+        {
+          id: 3,
+          organization_id: 1,
+          mmsi: 24.33,
+          name: "hyf-vessel3"
+        }
+      ]);
+    });
```

Comment:
Please add more realistic names to the vessels and add some more. You can use something like Mockaroo to generate fake data. Don't know if it does ship names :D

Reply:
Also, could you add some more organizations (again use Mockaroo - you can definitely get company names there) and then spread the vessels over all of the organizations.